r2/r2/controllers/api.py

```diff
@@ -885,6 +908,10 @@ def POST_unfriend(self, nuser, iuser, container, type):
         if new and type in self._sr_friend_types:
             ModAction.create(container, c.user, action, target=victim)
 
+        if was_mod:
+            send_mod_permission_message(container, c.user, victim,
+                                        was_mod_perms, demod=True)
```

Comment:
I don't understand, what is there to abuse? The sending of messages? There is a limit to how often permissions can be changed as well as how many mods can be added at a time. Not sure about demodding, but it gets balanced out by the other two?